app/src/main/java/protect/card_locker/ShortcutHelper.java

```diff
@@ -155,4 +155,26 @@ Bitmap createAdaptiveBitmap(@NotNull Bitmap in, int paddingColor) {
                 .setIntent(intent)
                 .setIcon(icon);
     }
+    static ShortcutInfoCompat.Builder createGroupShortcutBuilder(Context context, Group group) {
+        Intent intent = new Intent(context, MainActivity.class);
+        intent.setAction(Intent.ACTION_MAIN);
+        // Prevent instances of the view activity from piling up; if one exists let this
+        // one replace it.
+        intent.setFlags(intent.getFlags() | Intent.FLAG_ACTIVITY_SINGLE_TOP);
+        final Bundle bundle = new Bundle();
+        // Done like with the cards but with group._id (a string)
+        bundle.putString("id", group._id);
```

Comment:
Please use a more descriptive ID and make it a static final String in MainActivity to prevent typo risks. `id` is really the type of value that may be accidentally used for something else than the group ID one day.

Reply:
With the current syntax, I cannot set this string to be static in MainActivity :/ I would need to find out what to change to be able to make it a static variable